### DISCUSSION TITLE
Running in Docker Desktop on Mac not working

### DISCUSSION THREAD

**Discussion by gchattha** [koush/scrypted#433] [Q&A] [answered]
I've been using this software for a while on my Linux server in a Docker container and it's been great! I recently got an M1 Mac and moved my containers to there with all the same configuration but cameras keep showing "No Response" in the HomeKit app. The web page doesn't show up in "host" mode on Docker but works in "bridge" mode. From what I read, host mode is only supported on Linux - not Mac or Windows. My question is, how can I get this working in a Docker container under macOS like it was in Linux? Thanks @koush !

**Comment by koush** [comment, marked as answer]
you can't, this is documented somewhere on the wiki

**Comment by koush** [comment]
lots of ports.

**Reply by gchattha** [reply]
do you have a list of ports? maybe i can try forwarding them and seeing if it works?
